native/src/util.c

```diff
@@ -48,7 +48,7 @@ static char *joinHelper(char **in, size_t inlen, size_t inpos, size_t accum)
 {
   if (inpos == inlen)
   {
-    return strcpy(sqlite3_malloc(accum + 1) + accum, "");
+    return strcpy((char *)sqlite3_malloc(accum + 1) + accum, "");
```

Comment:
did this generate a warning or something on windows?

Reply:
An error actually, it didn't know how to cast the type